<launch>
    <arg name="qr_x" default="0.4" />
    <arg name="qr_y" default="0.2" />
    
    <node pkg="qr_code_pkg" type="qr_detector.py" name="qr_detector_node" output="screen">
     <param name="camera_topic_name" value="/realsense2/color/image_raw" />
     <param name="camera_info_topic_name" value="/realsense/color/camera_info" />
     <param name="output_topic_name" value="/qr_viewer" />

     <param name="qr_param_x" type="double" value="$(arg qr_x)" />
     <param name="qr_param_y" type="double" value="$(arg qr_y)" />
    </node>

    <node name="image_view_result" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/qr_viewer"/>
        <param name="autosize" value="true" />
    </node>
</launch>
